### PR TITLE
fix: use data-testid instead of data-test for element attributes

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

File: src/components/Checkout/OrderTotals.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@
 			</kv-tooltip>
 		</div>
 
-		<div class="order-total" data-test="total-due">
+		<div class="order-total" data-test="total-due" data-testid="total-due">
```
Comment (davidskiva) [inline src/components/Checkout/OrderTotals.vue:93]: Question.  why did you leave in the old data-test instead of just changing it.

Comment (mcstover) [reply src/components/Checkout/OrderTotals.vue:93]: Just didn't want to break anything in that case...

Comment (mcstover) [reply src/components/Checkout/OrderTotals.vue:93]: For that file in particular there are several of those as that was the original approach I took when refactoring the Checkout sections to Kiva Classic. I saw the versions without `id` and just made duplicates.

Comment (mcstover) [reply src/components/Checkout/OrderTotals.vue:93]: I'd be happy to make a sweep and remove these too though.

As for autolend settings which has a bunch of these, I'm going to create a separate task for that work.

Comment (davidskiva) [reply src/components/Checkout/OrderTotals.vue:93]: Okay, I'm fine with leaving as is.  @SqueakAttack you okay having as is?

**Review by davidskiva — APPROVED** [review]

File: src/components/Checkout/OrderTotals.vue (hunk 1 above)
Comment (SqueakAttack) [reply src/components/Checkout/OrderTotals.vue:93]: that's fine. if it might break something to change, maybe best to leave as is

Comment (mcstover) [reply src/components/Checkout/OrderTotals.vue]: Ok I'm removing them from the OrderTotals file. This won't affect anything in the Ui repo. Please look out for any issues around the totals in your robot tests. Everything removed has a `data-testid` now.

**Review by SqueakAttack — APPROVED** [review]
👍

**Pull request merged**